packages/core/src/checkout-buttons/strategies/braintree/braintree-venmo-button-strategy.ts

```diff
@@ -17,26 +17,42 @@ import {
     BraintreeTokenizePayload,
     BraintreeVenmoCheckout,
 } from '../../../payment/strategies/braintree';
+import { PaypalStyleOptions } from '../../../payment/strategies/paypal';
 import { CheckoutButtonInitializeOptions } from '../../checkout-button-options';
 import CheckoutButtonStrategy from '../checkout-button-strategy';
 import { CheckoutButtonMethodType } from '../index';
 
 import mapToLegacyBillingAddress from './map-to-legacy-billing-address';
 import mapToLegacyShippingAddress from './map-to-legacy-shipping-address';
 
-const venmoButtonStyle = {
-    backgroundColor: '#3D95CE',
-    backgroundPosition: '50% 50%',
-    backgroundSize: '80px auto',
-    backgroundImage: 'url("/app/assets/img/payment-providers/venmo-logo-white.svg")',
-    backgroundRepeat: 'no-repeat',
-    borderRadius: '4px',
-    cursor: 'pointer',
-    transition: '0.2s ease',
-    minHeight: '40px',
-    minWidth: '150px',
-    height: '100%',
-    width: '100%',
+const getVenmoButtonStyle = (styles: PaypalStyleOptions): Record<string, string> => {
+    const { color } = styles;
+
+    const colorParser = (c: string) => {
+        if (c === 'white') {
+            return '#FFFFFF';
+        }
+
+        return '#3D95CE';
+    };
+
+    return {
+        backgroundColor: colorParser(color || ''),
+        backgroundPosition: '50% 50%',
+        backgroundSize: '80px auto',
+        backgroundImage: `url("/app/assets/img/payment-providers/venmo-logo-${
+            color === 'white' ? 'blue' : 'white'
+        }.svg")`,
+        backgroundRepeat: 'no-repeat',
+        borderRadius: '4px',
+        cursor: 'pointer',
+        transition: '0.2s ease',
+        minHeight: '40px',
+        minWidth: '150px',
+        height: '100%',
+        width: '100%',
+        border: color === 'white' ? '1px solid black' : 'none',
```

Comment:
Should be able to use the value from enum here.

Reply:
Updated